zcash_test_vectors/zip_0244.py

```diff
@@ -138,7 +138,7 @@ def orchard_digest(tx):
         digest.update(orchard_actions_compact_digest(tx))
         digest.update(orchard_actions_memos_digest(tx))
         digest.update(orchard_actions_noncompact_digest(tx))
-        if tx.nVersionGroupId == NU7_VERSION_GROUP_ID:
+        if tx.version_bytes() == NU7_TX_VERSION | (1 << 31):
```

Comment:
this is better but error prone. need to compare to the correct static function

```python
if tx.version_bytes() == TransactionV5().version_bytes()
```
or similar for V6

Reply:
1. Importing TransactionZSA into this file will lead to a circular import error which I'd run into some weeks ago.
2. Initializing TransactionV5 or TransactionZSA will need a `rand` input, which will have to be imported, and might change the values of the generated test vectors again.

Reply:
ok, so let's store it in a constant and use it everywhere:
```python
NU5_VERSION_GROUP_ID = 0x26A7270A
NU5_TX_VERSION = 5
NU5_TX_VERSION_BYTES = NU5_TX_VERSION | (1 << 31)
```
(x2)

Reply:
including `def version_bytes()`

Reply:
Sounds good, done.